contrib/completion/zsh/_docker

```diff
@@ -2040,6 +2040,7 @@ __docker_service_subcommand() {
         (scale)
             _arguments $(__docker_arguments) \
                 $opts_help \
+                "($help -d --detach)"{-d=false,--detach=false}"[Disable detached mode]" \
```

Comment:
Boolean flag doesn't need `=false`.

Sorry for the delay, I have not a lot of time these days.